canvas_modules/common-canvas/src/common-properties/form/EditorForm.js

```diff
@@ -756,6 +762,23 @@ function _parameterValueLabels(parameter, l10nProvider) {
 	return [];
 }
 
+function _parameterValueDescription(parameter, l10nProvider) {
+	if (Array.isArray(parameter.getValidValues())) {
+		let key;
+		if (parameter.resource_key) {
+			key = parameter.resource_key;
+		} else {
+			key = parameter.name;
+		}
+		const paramLabels = [];
+		parameter.getValidValues().forEach(function(paramValue) {
+			paramLabels.push(l10nProvider.l10nValueLabel(key, String(paramValue)));
```

Comment:
This should be getting the desc and not label.

Reply:
Updated it with `l10nValueDesc`